v4l2loopback.c

```diff
@@ -2460,7 +2460,7 @@ static int v4l2_loopback_add(struct v4l2_loopback_config *conf, int *ret_nr)
 	nr = err;
 	err = -ENOMEM;
 
-	if (conf && conf->card_label && *(conf->card_label)) {
+	if (conf && *(conf->card_label)) {
```

Comment:
Or to be even a bit more explicit about the intended check:
```suggestion
	if (conf && conf->card_label[0]) {
```

Reply:
Adjusted, thanks!